src/api/schema/metrics/mod.rs

```diff
@@ -1,4 +1,9 @@
-use crate::event::{Event, Metric, MetricValue};
+mod bytes_processed;
```

Comment:
Moved existing metrics into a `metrics` sub-mod, to accommodate this growing area of the API schema